realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -353,6 +387,33 @@ public Builder(Context context) {
             initializeBuilder(context.getFilesDir());
         }
 
+        /**
+         * Creates an instance of the Builder for the RealmConfiguration.
+         *
+         * This will use the app's own internal directory for storing the Realm file. This does not require any
+         * additional permissions. The default location is {@code /data/data/<packagename>/files}, but can
+         * change depending on vendor implementations of Android.
+         *
+         * When opening the Realm for the first time, instead of creating an empty file,
+         * the Realm file will be copied from the provided assets file and used instead.
+         * WARNING: This could potentially be a lengthy operation so should ideally be done on a background thread.
+         *
+         * @param context Android application context.
+         * @param assetFile path to the asset database file.
+         */
+        public Builder(Context context, final String assetFile) {
```

Comment:
@beeender  From my perspective providing asset file without context is strange because we can get asset file only with App context. On the other hand we can provide `assetFile(context, assetFile)` or simply throw an exception in `assetFile()` if there is no provided context. What do you think?